RELEASE/scripts/autoscend/paths/community_service.ash

```diff
@@ -4942,11 +4938,11 @@ boolean cs_mpMaintain(){
 	} else if(my_maxmp() >= 120){
 		target = 120;
 	}
-	return cs_mpMaintain(target);
+	return acquireMP(target, 1500);
 }
 
 boolean cs_mpMaintain(int target){
-	return acquireMP(target, 1500, true);
+	return acquireMP(target, 1500);
 }
```

Comment:
is this still needed elsewhere?

Reply:
There are several spots in community service where we do `cs_mpMaintain(int)`
... however, thinking about it, this is just acting as a wrapper. I can replace `cs_mpMaintain(30)` with `acquireMP(30,1500)`

Only reason I have not yet is that I was thinking that the next step would be to actually remove the meatreserve (As we just plug in it an arbitrary value in several places to make it work, and if we do not then it disables using meat entirely) and also the allow free rest variable because there is no a single spot in the entire code where we have ever called on acquireMP or acquireHP and actually forbade free rests.